docs/challenges/language-support/ruby.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "Ruby"
+description: "Learn how to solve CodeCrafters challenges in Ruby"
+---
+
+<Note> **New to Ruby?** We recommend [Ruby in Twenty Minutes](https://www.ruby-lang.org/en/documentation/quickstart/) to get up to speed.</Note>
+
+# Local development
+
+To run the code for your challenge locally, you'll need `ruby` installed. Our test runners use version `3.3` (as of May 2024).
+
+The script for your challenge (like `./spawn_redis_server.sh`) will run the code using [Bundler](https://bundler.io/), a dependency manager which comes preinstalled with Ruby. It looks something like this:
```

Comment:
I don't think bundler doesn't come pre-installed with ruby, @glokta1 can you confirm if this is true?

Reply:
Ah, nice find! Back in the day we had to do `gem install bundler` 🙂 All good then!